lib/ttfunk/table/head.rb

```diff
@@ -22,10 +22,18 @@ class Head < TTFunk::Table
       attr_reader :glyph_data_format
 
       def self.encode(head, loca)
-        table = head.raw
-        table[8, 4] = "\0\0\0\0" # set checksum adjustment to 0 initially
-        table[-4, 2] = [loca[:type]].pack('n') # set index_to_loc_format
-        table
+        EncodedString.new do |table|
+          table << [head.version, head.font_revision].pack('N2')
+          table << Placeholder.new(:checksum, length: 4)
+          table << [
```

Comment:
I think, it would be OK to just chain shovels here.

```ruby
EncodedString.new << 
  [head.version, head.font_revision].pack('N2') <<
  Placeholder.new(:checksum, length: 4) <<
  [...].pack('Nn2q2n*')
```